.pre-commit-config.yaml

```diff
@@ -25,6 +25,6 @@ repos:
       - flake8-import-order
       name: "Check Code Style Using flake8"
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.812  # Use the sha / tag you want to point at
+    rev: v0.782  # Use the sha / tag you want to point at
```

Comment:
To bring it in line with the version number in `pyproject.toml`. Running the newer version locally causes it throw errors because it says it doesn't recognise any typing for `scipy` modules.